docs/src/increase-observability/integrate-observability/blackfire.md

```diff
@@ -29,7 +29,11 @@ All customers can also subscribe to Blackfire separately.
 
 ## Getting started
 
-### On a Grid plan or {{% names/dedicated-gen-3 %}} infrastructure
+### On a Grid plan
+
+By default, the [Blackfire automated integration](#automated-integration) is enabled on Grid plans. 
```

Comment:
:lady_beetle: It's only by default for people who have the Observability Suite. It's not available to other customers.